### DISCUSSION TITLE
Hash Rate is 0.0H/s. Why?

### DISCUSSION THREAD

**Discussion by webpinar** [BitMaker-hub/NerdMiner_v2#537] [Q&A]
Hello
I did everything well, bought my ESP32-WROOM-32, I setup the public pool, but it was working for me only 4 hours, later is the whole time only 0.0Hs. Any Help? I did several time to flash it again and reset, but nothing. Maybe to try another pool?. Thanks. Rgds
![Captura de pantalla 2024-12-08 a las 14 05 30](https://github.com/user-attachments/assets/954f87d3-4b85-47f1-b0cc-d2ae878d933a)

**Comment by sany3001** [comment]
Is your device well connected to Wi-Fi? Can you reach pool from your network?

**Reply by webpinar** [reply]
It look ok, Yesterday It was working untill I reboot the router. After it I can see on public web site my worker but not mining.

**Reply by webpinar** [reply]
Must be something with the port 21496, I have DECO TPLINK.

**Reply by webpinar** [reply]
![Captura de pantalla 2024-12-08 a las 18 26 20](https://github.com/user-attachments/assets/3844c774-bbe9-4765-9784-13c4d8ec0466)
I was trying different ESP cards, and the same.
